composer.json

```diff
@@ -22,28 +22,28 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
-        "jms/serializer": "^3.20",
-        "jms/metadata": "^2.5",
-        "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^3.4 || ^4.0 || ^5.0 || ^6.0"
+        "jms/serializer": "^3.28",
+        "jms/metadata": "^2.6",
+        "symfony/dependency-injection": "^3.4 || ^4.0 || ^5.0 || ^6.0 || ^7.0",
```

Comment:
it is crazy to see that we are able to run this bundle on a such large range of symfony versions